includes/templates/bootstrap/templates/tpl_shopping_cart_default.php

```diff
@@ -70,11 +70,11 @@
         <table id="shoppingCartDefault-cartTableDisplay" class="cartTableDisplay table table-bordered table-striped table-sm">
             <tr>
                 <th scope="col" id="cartTableDisplay-qtyHeading"><?php echo TABLE_HEADING_QUANTITY; ?></th>
-                <th scope="col" class="d-none d-sm-table-cell" id="cartTableDisplay-qtyUpdateHeading">&nbsp;</th>
+                <?php if (SHOW_SHOPPING_CART_UPDATE == 1 || SHOW_SHOPPING_CART_UPDATE == 3) { ?><th scope="col" class="d-none d-sm-table-cell" id="cartTableDisplay-qtyUpdateHeading"><span aria-label="<?php echo TEXT_CART_ARIA_HEADING_UPDATE_COLUMN; ?>">&nbsp;</span></th> <?php } ?>
```

Comment:
I'm bit bit more persnickety and don't like the 'inline' PHP conditionals.  Please indent the `<php` conditional to line up with its parent and render the HTML portion separately (also indented to match the other `th` elements).